databases/functions/userFollowUser/check-is-blurred-post.js

```diff
@@ -17,7 +17,7 @@ module.exports = async (userFollowUserModel, userIdFollower, _transaction = null
     raw: true
   });
 
-  if (signUserList.length <= MINIMUM_BLUR_USER_FOLLOWING) {
+  if (signUserList.length < MINIMUM_BLUR_USER_FOLLOWING) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `<=` to `<` in the conditional statement could potentially lead to a different behavior. If `signUserList.length` is equal to `MINIMUM_BLUR_USER_FOLLOWING`, `isBlurredPost` will not be set to `true`. Please ensure this is the intended behavior.
```diff
-  if (signUserList.length <= MINIMUM_BLUR_USER_FOLLOWING) {
+  if (signUserList.length < MINIMUM_BLUR_USER_FOLLOWING) {
     isBlurredPost = true;
   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->